Packet++/src/SingleCommandTextProtocol.cpp

```diff
@@ -143,8 +143,19 @@ namespace pcpp
 
 			// Remove XXX- and XXX<SP> since they are delimiters of the protocol where XXX is the usually status code
 			// Check RFC821 (SMTP) Section 3.3 and RFC959 (FTP) Section 4.2
-			std::regex reg(std::regex(getCommandInternal() + "-|" + getCommandInternal() + " "));
-			return std::regex_replace(option, reg, std::string());
+			std::vector<std::string> vDelim;
+			vDelim.push_back(getCommandInternal() + "-");
+			vDelim.push_back(getCommandInternal() + " ");
```

Comment:
nit: we could run `getCommandInternal()` only once and create + init the vector in one line:
```cpp
auto option = getCommandInternal();
auto vDelim = std::vector<std::string> { option + " ", option + "-"}
```